docs/agents/spec-process.md

```diff
@@ -0,0 +1,58 @@
+Every change that affects more than one agent should be initiated via a change to the specification rather than creating a cross-agent issue.
+
+## Advantages
```

Comment:
nit: I'd put the `Advantages` part at the end. I feel long term everyone will be interested in the process itself and not about the advantage of it.